From 2be197cb1ca475abd2777432028b550eb9a9cdf0 Mon Sep 17 00:00:00 2001
From: Liu Gang <Gang.Liu@nxp.com>
Date: Wed, 3 Feb 2016 14:57:11 +0800
Subject: [PATCH 137/151] layerscape/dts: Update GPIO node

commit 62a8d1cd36b53b69e1814f1f24821ae43cc88307
[Only picked ls1043a related content]

For ls1043a and ls2085a, the GPIO compatible should be
changed to "fsl,qoriq-gpio".

For ls2080a and ls2085a, the little-endian property
should be added since the GPIO register working on
little endian mode on the two platforms.

Signed-off-by: Liu Gang <Gang.Liu@nxp.com>
Integrated-by: Jilong Guo <jilong.guo@nxp.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1043a.dtsi | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1043a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1043a.dtsi
index d7d2fa6..15e3e4f 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1043a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1043a.dtsi
@@ -608,8 +608,8 @@
 			clocks = <&clockgen 4 0>;
 		};
 
-		gpio1: gpio@2300000 {
-			compatible = "fsl,ls1043a-gpio";
+		gpio0: gpio@2300000 {
+			compatible = "fsl,qoriq-gpio";
 			reg = <0x0 0x2300000 0x0 0x10000>;
 			interrupts = <0 66 0x4>;
 			gpio-controller;
@@ -618,8 +618,8 @@
 			#interrupt-cells = <2>;
 		};
 
-		gpio2: gpio@2310000 {
-			compatible = "fsl,ls1043a-gpio";
+		gpio1: gpio@2310000 {
+			compatible = "fsl,qoriq-gpio";
 			reg = <0x0 0x2310000 0x0 0x10000>;
 			interrupts = <0 67 0x4>;
 			gpio-controller;
@@ -628,8 +628,8 @@
 			#interrupt-cells = <2>;
 		};
 
-		gpio3: gpio@2320000 {
-			compatible = "fsl,ls1043a-gpio";
+		gpio2: gpio@2320000 {
+			compatible = "fsl,qoriq-gpio";
 			reg = <0x0 0x2320000 0x0 0x10000>;
 			interrupts = <0 68 0x4>;
 			gpio-controller;
@@ -638,8 +638,8 @@
 			#interrupt-cells = <2>;
 		};
 
-		gpio4: gpio@2330000 {
-			compatible = "fsl,ls1043a-gpio";
+		gpio3: gpio@2330000 {
+			compatible = "fsl,qoriq-gpio";
 			reg = <0x0 0x2330000 0x0 0x10000>;
 			interrupts = <0 134 0x4>;
 			gpio-controller;
-- 
2.1.0.27.g96db324

